proto/Cargo.toml

```diff
@@ -55,6 +55,7 @@ idna = "^0.1.4"
 lazy_static = "^1.0"
 log = "^0.3.5"
 openssl = { version = "^0.9.8", features = ["v102", "v110"], optional = true }
+radix_trie = "0.1.2"
```

Comment:
This is being put in proto so that mean that you are planning for the resolver to use the trie. However, does resolution really require this? This seems like server- and maybe (non-resolver) client- specific stuff.

Reply:
I was thinking it would be generally usable across libraries. There are usage rules for resolvers, clients and servers alike.

I could move it to an independent crate.